assets/component-image-with-text.css

```diff
@@ -321,9 +321,40 @@
   margin-bottom: 1rem;
 }
 
+@media screen and (max-width: 749px) {
+  .collapse-padding .image-with-text__grid .image-with-text__content {
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .collapse-padding .image-with-text__grid:not(.image-with-text__grid--reverse) .image-with-text__content {
+    padding-right: 0;
+  }
+
+  .collapse-padding .image-with-text__grid--reverse .image-with-text__content {
+    padding-left: 0;
+  }
+}
+
 /* check for flexbox gap in older Safari versions */
 @supports not (inset: 10px) {
   .image-with-text .grid {
     margin-left: 0;
   }
 }
+
+/*
+  Multirow
+  note: remove from this stylesheet if multirow-specific styles increase signficantly
+*/
+.multirow .image-with-text {
+  margin-bottom: var(--grid-mobile-vertical-spacing)
```

Comment:
> I think here we should maybe not apply that to the last child.

Whoops, did I not handle that? I honestly forget why I decided margins would be simpler. But I can reconsider 👍 

> `<div class="multirow page-width">`

heh, I _really_ don't like this pattern 😄 Even if we weren't adding a section background color now, the possibility that we ever would add that or a fullwidth layout option in the future is enough to include a more robust dom structure from the start. The minimum-viable-markup approach we've historically taken has resulted in a bunch of `multirow-wrapper` type retrofits and generally inconsistent markup between sections.

Reply:
I remember why i didn't use flex. Just that it would require I name the page-width div to style it sensibly. I somewhat want to propose a standard section structure for the theme where this particular element would inherently have a name like `section__inner` (as in `.multirow.section__outer > .section__inner`), but for now I just extended the multirow namespace.